packages/pkp-base/src/lib/pkp-base.ts

```diff
@@ -255,6 +255,75 @@ export class PKPBase<T = PKPBaseDefaultParams> {
     }
   }
 
+  private validateAuthenticate() {
+    const providedAuthentications = [
+      this.controllerAuthSig,
+      this.authContext,
+      this.controllerSessionSigs,
+    ].filter(Boolean).length;
+
+    if (providedAuthentications !== 2) {
+      // log which authentications has the user provided
+      if (this.controllerAuthSig) {
```

Comment:
Note for myself (@Ansonhkg): remove this for 0.1 release https://linear.app/litprotocol/issue/LIT-3032/[auth-unification]-remove-all-v0-stuff-before-v1-release